### DISCUSSION TITLE
[Feature] Album descriptions in mobile app

### DISCUSSION THREAD

**Discussion by Xmoe** [immich-app/immich#9372] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Currently you neither view nor edit album descriptions in the mobile app on Android. This feature exists in the web version.
Adding a text field to the top of the album view to bring it up to parity with the web version would be great :)

### Versions:
Server: `1.103.1`
App: `1.103.1 build 137` (on Android 13)

### Related
https://github.com/immich-app/immich/discussions/2490 (notably the feature request included the app)
https://github.com/immich-app/immich/pull/3558

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by sannidhyaroy** [comment]
+1. I'd love to see it as well...

**Comment by Danie10** [comment]
I've been looking for this too as I share quite a few albums publicly and with friends, and the description sets the context for what the album is about.

**Comment by JobiJoba** [comment]
I'll look at that :) I was looking for this also

**Reply by JobiJoba** [reply]
Added a new PR for this feature #18886

**Comment by bo0tzz** [comment]
This has been in for a while now
